### ISSUE TITLE
Error AutoDrivePathFinder.lua

### ISSUE THREAD

**Issue by 820fahrer** [Stephan-S/FS19_AutoDrive#803]
Hey guys,

I found another issue. When activating a driver on our server an error occures:
2019-11-06 18:03 Error: Running LUA method 'update'.
2019-11-06 18:03 C://gameservers//s1335822//ls2019//gameProfile/mods/FS19_AutoDrive/scripts/AutoDrivePathFinder.lua:1147: attempt to call method 'BoxesIntersect' (a nil value)

I don't know why this happens. I've recalculated all routes before starting the driver.

I've downloaded the latest version of AutoDrive before starting the game.

**Comment by Roby1164** [comment]
I confirm error

**Comment by worker2419** [comment]
same problem here. lastest Vision from zip (...step 3) error occurred when ad driver start to unload combine harvester
